### DISCUSSION TITLE
hands down layout - missing in the article about using a thumb for a letter

### DISCUSSION THREAD

**Discussion by rainymatch** [precondition/precondition.github.io#20] [Ideas]
hey Precondition,

thx for the wonderful articles on the home row mods and the "E-thumb" issue. thanks to you I discovered the KMond software which completely changed the way I am using the keyboard of my laptop. and i freakin love it

you are encouraging the readers to get in touch if they know about any other layouts using the 2nd thumb for a letter, however I could not find any email address of yours or such, so here I am, writing in the github discussion

hands down layout
https://sites.google.com/alanreiser.com/handsdown#h.holg42y4ug33

I have developped my own layout, taking quiet some inspiration from the hands down approach and can confirm that consonants feel much better on the thumb. E felt weird, N was fine but i decided to go for T in the end basically just because it felt a bit better. And it is absolutely great.

cheers, 
Martin
hyca.martin@gmail.com

**Comment by precondition** [comment]
Hello,
I did consider adding the Hands Down variations which feature a letter on thumb but I think it's wiser to first wait for a settled layout.
From the website:
> The result of this review will certainly be very close to what is here, but some slight changes could be made to the layouts themselves, so it might be best to consider everything here as a very late stage beta version.
